conda_smithy/feedstock_content/ci_support/checkout_merge_commit.sh

```diff
@@ -23,6 +23,5 @@ fi
 # Check for merge conflicts.
 if [[ -n "${CIRCLE_PR_NUMBER}" ]]
 then
-    git checkout -qf "pr/${CIRCLE_PR_NUMBER}/merge"
```

Comment:
Also re-tested with this change and it still [works]( https://github.com/conda-forge/freeglut-feedstock/pull/6#issuecomment-250491357 ).